src/ruby/yast/debugger.rb

```diff
@@ -73,6 +95,17 @@ def installed?
 
     private
 
+      # read the debugger value from Y2DEBUGGER environment variable,
+      # do case insensitive match
+      # @return [String,nil] environment value or nil if not defined
+      def read_dbg_env
```

Comment:
I think simple `env_value` is enough for this method. Read connect in my mind with something expensive, which is not this case.